src/audio/codec_adapter/codec/dcblock.c

```diff
@@ -0,0 +1,361 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2020 Intel Corporation. All rights reserved.
+//
+// Author: Johny Lin <johnylin@google.com>
+
+#include <ipc/stream.h>
+#include <sof/audio/codec_adapter/codec/generic.h>
+#include <sof/audio/codec_adapter/codec/dcblock.h>
+#include <sof/audio/format.h>
+
+/*****************************************************************************/
+/* DCBlock processing functions						     */
+/*****************************************************************************/
+
+static int32_t dcblock_generic(struct dcblock_state *state,
```

Comment:
should this be inline ?

Reply:
if it does not contain the loop then yes

Reply:
this function will be contained in loops

Reply:
then yep lets inline it